packages/integration-tests/src/playbook/contracts/Middle.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+interface IGreeter {
+    function greet() external view returns (string memory);
+    function setGreeting(string memory) external;
+}
+
+contract Middle {
```

Comment:
Why is the file and contract name Middle?

Reply:
This is just the name of the contract that we use in the test suite. The suite contains 3 contracts Root - Middle - Caller